client/my-sites/site-settings/disconnect-site/too-difficult.jsx

```diff
@@ -0,0 +1,70 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get, map, pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import QuerySitePlans from 'components/data/query-site-plans';
+import SectionHeader from 'components/section-header';
+import { getSitePlanSlug } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getPlanClass } from 'lib/plans/constants';
+
+function getFeatures( planClass, translate ) {
+	const features = {
+		backups: translate( 'Backups' ),
+		security: translate( 'Security Scanning' ),
+		antispam: translate( 'Antispam' ),
+		stats: translate( 'Stats' ),
+		publicize: translate( 'Publicize' ),
+		subscriptions: translate( 'Subscriptions' ),
+		other: translate( 'Other' ),
+	};
+	return get(
```

Comment:
As mentioned in my previous review, free plan will need some features to show. Have a look around `lib/plans/constants`, there may be something reusable there.